### DISCUSSION TITLE
(neuer) HmIP-STH wird in HA nicht angezeigt

### DISCUSSION THREAD

**Discussion by MichaelKreitmann** [SukramJ/aiohomematic#1130] [Q&A] [answered]
Hallo!
Ich habe eine CCU2 mit einigen Komponenten, die ich alle problemlos in HA sehen und anzeigen kann. Jetzt habe ich eine HmIP-STH dazugekauft und dieser wird in der CCU2 auch problemlos angezeigt. Nur bekomme ich das Teil nicht in HA ... ehabe schon versucht alles zu deinstallieren usw. - nicht. Ich sehe in HA alles - aber nur diesen Sensor nicht ....
Muss ich da noch irgendwas spezielles machen weil es ein "IP" Gerät ist, oder wie kann ich die Liste der Homematic Geräte in HA aktulisieren .. ?
Vielen Dank für eure Hilfe!

Gruß
Michael

**Comment by SukramJ** [comment]
Mein HmIP-STH wird angezeigt.
Welche Versionen (HA, Integration, CCU) verwendest Du? 
Hänge bitte die Diagnoseinformation der Integration an.

**Reply by MichaelKreitmann** [reply]
Danke für deine Antwort.
Leider weiß ich im Moment nicht, wie ich an die Diagnosedaten komme, daher:
Integration: 1.39.2
Home Assistant 2023.7.3
Supervisor 2023.07.1
Operating System 10.3
Frontend 20230705.1 - latest
CCU2: 2.61.7
Ich hoffe das hilft ?

**Reply by SukramJ** [reply]
> Leider weiß ich im Moment nicht, wie ich an die Diagnosedaten komme, daher:

Du klickst auf die Integration, danach auf die 3 Punkte neben deiner Instanz und dann Diagnoseinformationen herunterladen.

**Reply by MichaelKreitmann** [reply]
{
  "home_assistant": {
    "installation_type": "Home Assistant OS",
    "version": "2023.7.3",
    "dev": false,
    "hassio": true,
    "virtualenv": false,
    "python_version": "3.11.4",
    "docker": true,
    "arch": "aarch64",
    "timezone": "Europe/Berlin",
    "os_name": "Linux",
    "os_version": "6.1.21-v8",
    "supervisor": "2023.07.1",
    "host_os": "Home Assistant OS 10.3",
    "docker_version": "23.0.6",
    "chassis": "embedded",
    "run_as_root": true
  },
  "custom_components": {
    "huawei_solar": {
      "version": "1.2.6",
      "requirements": [
        "huawei-solar==2.2.8"
      ]
    },
    "hacs": {
      "version": "1.32.1",
      "requirements": [
        "aiogithubapi>=22.10.1"
      ]
    },
    "homematicip_local": {
      "version": "1.39.2",
      "requirements": [
        "hahomematic==2023.7.3"
      ]
    }
  },
  "integration_manifest": {
    "domain": "homematicip_local",
    "name": "Homematic(IP) Local",
    "codeowners": [
      "@danielperna84",
      "@SukramJ"
    ],
    "config_flow": true,
    "dependencies": [],
    "documentation": "https://github.com/danielperna84/custom_homematic",
    "homekit": {},
    "integration_type": "hub",
    "iot_class": "local_push",
    "issue_tracker": "https://github.com/danielperna84/hahomematic/issues",
    "loggers": [
      "hahomematic"
    ],
    "requirements": [
      "hahomematic==2023.7.3"
    ],
    "ssdp": [
      {
        "manufacturer": "EQ3",
        "manufacturerURL": "http://www.homematic.com"
      }
    ],
    "version": "1.39.2",
    "zeroconf": [],
    "is_built_in": false
  },
  "data": {
    "config": {
      "entry_id": "78df5f01cc3df6faca0f02ffc5d587b9",
      "version": 2,
      "domain": "homematicip_local",
      "title": "Homematic CCU2",
      "data": {
        "instance_name": "Homematic CCU2",
        "host": "192.168.178.35",
        "username": "**REDACTED**",
        "password": "**REDACTED**",
        "tls": false,
        "verify_tls": false,
        "sysvar_scan_enabled": true,
        "sysvar_scan_interval": 30,
        "callback_host": null,
        "callback_port": null,
        "json_port": null,
        "enable_system_notifications": true,
        "interface": {
          "HmIP-RF": {
            "port": 2010
          },
          "BidCos-RF": {
            "port": 2001
          }
        }
      },
      "options": {},
      "pref_disable_new_entities": false,
      "pref_disable_polling": false,
      "source": "user",
      "unique_id": "LEQ1273564",
      "disabled_by": null
    },
    "platform_stats": {
      "binary_sensor": 32,
      "climate": 13,
      "sensor": 7,
      "switch": 1
    },
    "devices": [
      "HM-CC-RT-DN",
      "HM-ES-PMSw1-Pl",
      "HM-Sec-SCo",
      "HM-TC-IT-WM-W-EU",
      "HM-WDS10-TH-O",
      "HmIP-STH"
    ]
  }
}

**Reply by MichaelKreitmann** [reply]
Hmm, also da ist das Teil ausgelistet ...

**Reply by MichaelKreitmann** [reply]
Edit: Es geht !! Jetzt ist das Teil auch da - keine Ahnung .. aber - egal: Vielen Dank für deine Hilfe! Dankeschön!!

Kann ich die "verstecken" Entitäten (Temperatur und Luftfeuchtigkeit) auch sichtbar machen ? (ich brauche von dem Teil nur die Temperatur)

Danke dir!

**Reply by SukramJ** [reply, marked as answer]
Die must die Entitäten aktivieren. Das steht auch in der Doku.
